site/src/lib/blocks.ts

```diff
@@ -85,6 +87,7 @@ export const readBlocksFromDisk = (): ExpandedBlockMetadata[] => {
 
       const metadata: ExpandedBlockMetadata = {
         // @todo should be redundant to block's package.json#name
+        componentId: `${FRONTEND_URL}/blocks/${packagePath}`,
```

Comment:
I'm generating the `componentId` at build-time, right where we're inserting the `packagePath` as well. @CiaranMn lmk if you would prefer to have this in the API instead.

@nathggns I'm presently generating the `componentId` in a format which matches the current entityStore cache. Lmk if we can use any other unique property, eg, the `name` of the block, directly instead.